app/Http/Controllers/Admin/LocationController.php

```diff
@@ -744,7 +744,7 @@ public function importLocationsAction(ServerRequestInterface $request): Response
                 $added++;
             }
 
-            if (!$exists || $options === 'update') {
+            if (!$exists || stripos($options, 'update') !== false) {
```

Comment:
Does `$options` have a fixed list of possible values, or is it a completely variable string that might contain anything.

If the former, then it would be clearer to write `!$exists || $options === 'update' || $options === 'update2' || $options === 'do-update'`